int32 turtle_id
string sprite_name
---
bool success





